### PR TITLE
#smtpemail

### PR DESCRIPTION
**Pull request by visu231**

### CODE REVIEW

File: visu/smtpemail/routes/action.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+});
+var mailOptions, host, link;
+router.post('/signup', urlencoded, async(req, res) => {
+    {
+        try {
+            const email = req.body.email;
+
+            function validateEmail(email) {
+                var emailPattern = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,4}$/;
+                return emailPattern.test(email);
```
Comment (pbravichandhar) [inline visu/smtpemail/routes/action.js]: where is arrow function

Comment (pbravichandhar) [reply visu/smtpemail/routes/action.js]: Why are you using VAR

File: visu/smtpemail/routes/action.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            }
+            const value = (validateEmail(email));
+            if (value) {
+                console.log("jii")
```
Comment (pbravichandhar) [inline visu/smtpemail/routes/action.js]: Console ?

File: visu/smtpemail/routes/action.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+                host = req.get('host');
+                link = "http://" + req.get('host') + "/students/verify?email=" + email + "&id=" + secretToken;
+                console.log(secretToken);
+                mailOptions = {
+                    to: email,
+                    subject: "Please confirm your Email account",
+                    html: "Hello,<br> Please Click on the link to verify your email.<br><a href=" + link + ">Click here to verify</a>"
+                }
+                console.log(mailOptions);
+                smtpTransport.sendMail(mailOptions, function(error, response) {
```
Comment (pbravichandhar) [inline visu/smtpemail/routes/action.js]: can you use promise or asyn await for this thing ... surf in google

File: visu/smtpemail/server.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+require('dotenv/config');
+app.use(bodyparser.json());
+
+const studentrouter = require('./routes/action');
+app.use('/students', studentrouter);
+
+
+//connect the database with the node 
+mongoose.connect(url, { useNewUrlParser: true }, async() => {
+    console.log("Database Connected");
```
Comment (pbravichandhar) [inline visu/smtpemail/server.js]: what is the use of async here ?

**Comment by mukeshkannanai** [comment]
.

**Pull request merged**